src/components/Search/SearchResultsListItem.tsx

```diff
@@ -125,7 +142,19 @@ function SearchResultsListItem({
                       className='time-icon'
                       fontSize='small'
                     />
-                    {String(dayjs(itemDate).format('MM/DD/YY HH:mm'))}
+
+                    {/* Tooltip with UTC time */}
+                    <Tooltip title={`${utcTime}`}>
+                      <span>
+                        {date}{' '}
+                        <span style={{ fontWeight: 'bold' }}>
```

Comment:
not sure why you changed the style, I think it's not needed. Then you can remove the spans as well